pkg/repository/repository.go

```diff
@@ -669,3 +676,74 @@ func (r *repository) transpileFilter(filter filtering.Filter) (*clause.Expr, err
 		filter: filter,
 	}).Transpile()
 }
+
+func (r *repository) ListModelTriggers(ctx context.Context, pageSize int64, page int64, order ordering.OrderBy, modelUID *string, startTimeFrom *time.Time, startTimeTo *time.Time) (modelTriggers []*datamodel.ModelTrigger, totalSize int64, err error) {
+	logger, _ := custom_logger.GetZapLogger(ctx)
+
+	var whereConditions []string
+	var whereArgs []any
+	if modelUID != nil {
+		whereConditions = append(whereConditions, "model_uid = ?")
+		whereArgs = append(whereArgs, *modelUID)
+	}
+	if startTimeFrom != nil {
+		whereConditions = append(whereConditions, "start_time >= ?")
+		whereArgs = append(whereArgs, *startTimeFrom)
+	}
+	if startTimeTo != nil {
+		whereConditions = append(whereConditions, "start_time <= ?")
+		whereArgs = append(whereArgs, *startTimeTo)
+	}
+
```

Comment:
I'm missing a condition on the requester (for the cases where the list is requested by a user that doesn't own the model)

Reply:
will confirm the requirements and fix in next PR